inscriptions/ISic000276.xml

```diff
@@ -117,7 +121,7 @@
                         </origin>
                         <provenance type="found">Original discovery not recorded.</provenance>
                         <provenance type="observed" subtype="autopsied">None</provenance>
-                        <acquisition/>
+                        <acquisition>Messina, Museo regionale interdisciplinare di Messina no. A3561</acquisition>
```

Comment:
not necessary to fill this one in at this point, this would be for more specific acquisition history, rather than simply current location which is encoded in repository already.